sergJess/React-RS-2023
# React components

1) Task: https://github.com/rolling-scopes-school/tasks/tree/master/react/modules/module01
2) ScreenShot: 
![React-done](https://user-images.githubusercontent.com/77243210/226199669-919b812d-adaf-465b-aaf2-36ca3d3e00e2.png)
3) Done: 19/03/2023 / deadline 20/03/2023
4) Score: 10 / 15 - I'm not sure that i made all right way
1) TS is used with React and Vite in this project
2) Added Eslint and Prettier. 
3) Added React-Router version 6. Router works
4) Search bar and cards exist and they work
5) Test use
Note: project should be run with npm run dev - not by build command(may occur error and router will not work)
What should be added or improved later in the next branches:
1) add absolute paths in bundle - i tried but it right now does not work - used rel paths
2) add more different tests
3) improve config of eslint and prettier
4) add footer and improve adaptive styles for app
